packages/graphql-hooks/src/useClientRequest.js

```diff
@@ -1,5 +1,5 @@
-import React from 'react'
 import { dequal } from 'dequal'
+import React from 'react'
```

Comment:
I wonder how this would work before without this import!

Reply:
I believe that my "fix on save" just swapped the order of the imports and the GitHub is not the diff properly.